Files/BaseLayout.cs

```diff
@@ -413,7 +414,7 @@ protected override async void OnNavigatedTo(NavigationEventArgs eventArgs)
                 await ParentShellPageInstance.FilesystemViewModel.SetWorkingDirectoryAsync(navigationArguments.NavPathParam);
 
                 // pathRoot will be empty on recycle bin path
-                var workingDir = ParentShellPageInstance.FilesystemViewModel.WorkingDirectory;
+                var workingDir = ParentShellPageInstance.FilesystemViewModel.WorkingDirectory ?? string.Empty;
```

Comment:
Could not reproduce.

Reply:
This one will actually do nothing since `GetPathRoot()` has a failsafe for null or empty strings, and we also check below if the `pathRoot` is null or empty.